pkg/manager/manager_test.go

```diff
@@ -470,8 +471,10 @@ var _ = Describe("manger.Manager", func() {
 				Expect(ok).To(BeTrue())
 				multilock, isMultiLock := cm.resourceLock.(*resourcelock.MultiLock)
 				Expect(isMultiLock).To(BeTrue())
-				_, primaryIsConfigMapLock := multilock.Primary.(*resourcelock.ConfigMapLock)
```

Comment:
This type is no longer exported in 1.24. So I replaced it with closest equivalent code I could think of